CHANGELOG.rst

```diff
@@ -1,5 +1,59 @@
+
+The project status and the indexing progresses are now available in the project settings tab.
+
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**🌟 New Features**
+
+- 🐳️ **CLI**: Pass docker run args to session start (`#3487 <https://github.com/SwissDataScienceCenter/renku-python/issues/3487>`_).
+- 👩‍💻️ **CLI**: Shell completion for sessions (`#3450 <https://github.com/SwissDataScienceCenter/renku-python/issues/3450>`_).
+- 📎️ **CLI**: Export dataset keywords (`#3454 <https://github.com/SwissDataScienceCenter/renku-python/issues/3454>`_).
+- **KG**: Dataset Details API response enhanced with info about project specific dataset identifier (`#1546 <https://github.com/SwissDataScienceCenter/renku-graph/issues/1546>`_).
+- **KG**: ``slug`` as a copy of ``name`` on responses from all Dataset APIs (`#1544 <https://github.com/SwissDataScienceCenter/renku-graph/issues/1544>`_).
+
+**✨ Improvements**
+
+- 👩‍💻️ **UI**: Major rework and simplification of project status page (moved to Settings -> General) (`#2315 <https://github.com/SwissDataScienceCenter/renku-ui/issues/2315>`_).
+- 🗑️ **UI**: Add a delete project button on the settings tab (`#2416 <https://github.com/SwissDataScienceCenter/renku-ui/issues/2416>`_).
+
+**🐞 Bug Fixes**
+
+- 📎️ **CLI**: Fixed dataset update with external files (`#3379 <https://github.com/SwissDataScienceCenter/renku-python/issues/3379>`_).
+- 🛠️ **CLI**: Fixed special paths in workflow files and bump ``toil/cwltool`` (`#3489 <https://github.com/SwissDataScienceCenter/renku-python/issues/3489>`_).
+- 🖼️ **UI**: Made text not selectable in entity cards (`#2546 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2546>`_).
+- 📎️ **UI**: Fixed the "add dataset to project" feature (`#2549 <https://github.com/SwissDataScienceCenter/renku-ui/pull/2549>`_).
+- 🔗️ **UI**: Fixed clicking on a file name on the dataset view wrongly leading to lineage and not contents (`#1270 <https://github.com/SwissDataScienceCenter/renku-ui/issues/1270>`_).
+- 🔗️ **UI**: Fixed getting a 404 page when switching from lineage view to contents (`#2571 <https://github.com/SwissDataScienceCenter/renku-ui/issues/2571>`_).
+
+Internal Changes
+~~~~~~~~~~~~~~~~
+
+**✨ Improvements**
+
+- 🛠️ **Gateway**: add new path to knowledge-graph webhooks
+  (`#639 <https://github.com/SwissDataScienceCenter/renku-gateway/issues/639>`_).
+
+**🐞 Bug Fixes**
+
+- 🛠️ **Core**: Fixed working on branches in the ``core-svc`` (`#3472 <https://github.com/SwissDataScienceCenter/renku-python/issues/3472>`_).
+- 🛠️ **Core:**: Return proper errors on migrations check (`#3334 <https://github.com/SwissDataScienceCenter/renku-python/issues/3334>`_).
+- 🛠️ **Gateway**: Fixed logout with new Keycloak version (`#655 <https://github.com/SwissDataScienceCenter/renku-gateway/issues/655>`_).
+
+Individual components
+~~~~~~~~~~~~~~~~~~~~~~
+
+- `renku-gateway 0.20.1 <https://github.com/SwissDataScienceCenter/renku-gateway/releases/tag/0.20.1>`_
```

Comment:
Please remove this and all mentions of the gateway for this release. Version `0.20.1` of the gateway was already merged with the keycloak upgrade (renku version `0.27.0`). We needed a fix on the gateway for the keycloak upgrade because the keycloak API changed.

Reply:
Ok done